src/repositories/searchResult.ts

```diff
@@ -0,0 +1,30 @@
+import { Prisma } from '@prisma/client';
+
+import dbClient from '@/config/database';
+
+const searchResultScraping = {
+  keyword: true,
+  userId: true,
+};
+
+export type SearchResultScraping = Prisma.SearchResultGetPayload<{
+  select: typeof searchResultScraping;
+}>;
+
+export const createSearchResult = async (
+  searchResultAttributes: Prisma.SearchResultCreateManyInput
+) => dbClient.searchResult.createMany({ data: searchResultAttributes });
+
+export const updateSearchResult = async (args: Prisma.SearchResultUpdateManyArgs) =>
+  dbClient.searchResult.updateMany(args);
+
+export const listSearchResultForScraping = async (userId: number) => {
```

Comment:
This method name is a bit inconsistent (the suffix `ForScraping` is added only this method and seems redundant):

```suggestion
export const listSearchResult = async (userId: number) => {
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/44/commits/772649044e477fe0425328fbaae9c5acd0eccb73